rpa-builder/6.1/modules/ROOT/pages/_partials/toolbox-web-automation-web-session.adoc

```diff
@@ -128,27 +129,39 @@ automatically.
 +
 The Workflow Run ends in an error.
 
-The standard response of the browser and the execution are defined in
-the *General Settings* area.
+Define the standard response of the browser and the execution in the *General Settings* section.
 
-You use the *Initial URL* to specify the initial URL of the web page you
-want to call.
+=== General Settings 
 
+image:toolbox-variable-web-automation-web-session-general-settings.png[50%, 50%, The General Settings section of the Wizard]
+
+* *Initial URL*
++
+Specifies the initial URL of the web page you
+want to call.
++
 If another web session that was created with the same web session ID is
 opened, the initial URL from that session is ignored.
 
-*Open Browser Wizard when done*: If this option is enabled, the Browser
-Wizard is opened when the Web Session Wizard is closed.
-
-*Keep out mouse, move it to*: In some cases, the mouse could hide
-certain buttons or texts. You use this option to define where you want
-the mouse to move to:
-
-image:toolbox-variable-web-automation-web-session-images/image4.png["An image showing the Keep out mouse, move it to: option"]
+* *Open Browser Wizard when done*
++
+If this option is enabled, the Browser
+Wizard opens when the Web Session Wizard is closed.
 
+* *Keep out mouse, move it to*
++ 
+Use this option to define where you want the mouse to move to. This prevents the mouse from covering certain buttons or texts.
++
+image:toolbox-variable-web-automation-web-session-images/image4.png[50%, 50%, "An image showing the Keep out mouse, move it to: option"]
```

Comment:
Though it is an alt text, it would be great to add the full stop at the end of the sentence to improve the pronunciation of the speech machine.